bin/miden-tx-prover/Cargo.toml

```diff
@@ -0,0 +1,55 @@
+[package]
+name = "miden-tx-prover"
+version = "0.6.0"
+description = "Miden rollup transaction prover"
+readme = "README.md"
+categories = ["no-std"]
+keywords = ["miden", "transaction"]
+license.workspace = true
+authors.workspace = true
+homepage.workspace = true
+repository.workspace = true
+rust-version.workspace = true
+edition.workspace = true
+
+[[bin]]
+name = "miden-tx-prover-service"
+path = "src/bin.rs"
+
+[features]
+default = ["std"]
+std = []
+testing = ["miden-objects/testing", "miden-lib/testing", "miden-tx/testing"]
+async = []
+
+[dependencies]
+miden-lib = { workspace = true }
+miden-objects = { workspace = true }
+miden-prover = { workspace = true }
+miden-verifier = { workspace = true }
+miden-tx = { workspace = true }
```

Comment:
I think `async` could be used here independently of user-facing features (and then we could also remove the above `async`).

Reply:
Closing as this is pretty similar to https://github.com/0xPolygonMiden/miden-base/pull/881#discussion_r1769049854